package.json

```diff
@@ -23,8 +24,15 @@
   },
   "homepage": "https://github.com/Soma-Technologies-Inc/visitAfrica-backend#readme",
   "devDependencies": {
```

Comment:
why babel in devDependencies?

Reply:
they should be in Dependencies but i solved it